nc-news-react/src/components/Post.jsx

```diff
@@ -0,0 +1,21 @@
+function Post () {
```

Comment:
Again, this is not specific to the ticket, so I'm not going to feed back on it as part of the PR. You'll need to include this again once you reach the relevant ticket.